tools/multicluster/verify.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Verify the signature of a binary with the operator's public key
+# goreleaser takes care of calling this script as a hook.
+
+ARTIFACT=$1
+SIGNATURE="${ARTIFACT}.sig"
+
+HOSTED_SIGN_PUBKEY="https://cosign.mongodb.com/mongodb-enterprise-kubernetes-operator.pem" # to complete
```

Comment:
can this change? Should that be hidden or public? Background: should we move it to evergreen vars?

Reply:
This stores our public key so that anyone can verify that the binary was signed by us. IMO it doesn't need to be hidden. It's also a nice public example on how to verify signatures